jxl/src/headers/frame_header.rs

```diff
@@ -220,6 +220,30 @@ struct RestorationFilter {
     extensions: Extensions,
 }
 
+#[derive(UnconditionalCoder, Debug, PartialEq)]
+pub struct Permutation {}
+
+pub struct TocNonserialized {
+    pub permuted: bool,
+    pub num_entries: u32,
+    pub entries: Vec<u32>,
```

Comment:
I'm not entirely sure why this field is here...

Reply:
I thought all entries in the serialized struct needed to be mirrored in the nonserialized?

Reply:
IIRC the nonserialized fields are meant to be inputs for the parser that do not come from the bitstream, so only the things that are needed from outside should be there.

Reply:
it it gone now